pages/api/funding/balanceOnDate.ts

```diff
@@ -37,93 +36,92 @@ export default async function handler(
       creationDate: {
         lt: firstThisMonth.toISOString(),
       },
+      name: { not: "OPEN_SOURCE" },
     },
-    distinct: ["address"],
+    distinct: ["address", "name"],
```

Comment:
Mm, by adding this won't you get a result back for every possible name @Riglanto ?

I.e. it's possible to get

EXTREME_POVERTY $ADDR$ 100% choiceNum=1 
OPEN_SOURCE $ADDR$ 50% choiceNum=8
ENDAOMENT $ADDR$ 50% choiceNum=8
CLIMATE $ADDR$ 100% choiceNum=4

Which would result in double counting of your balance further down the calcs I think?